services/addressService.ts

```diff
@@ -307,3 +307,11 @@ export async function fetchAddressWithTxsAndPrices (addressString: string): Prom
   }
   return result
 }
+
+export async function fetchAddressesByPaybuttonId(paybuttonId: string): Promise<string[]> {
```

Comment:
Could you refactor it so:
1. It doesn't uses `then`, only the `await` in parenthesis (we stick to this pattern of only `async/await` across the code) 
2. It doesn't uses the `Address.id` but the `Address.address`. Truth is this was implemented badly by me: there is absolutely no need for the address to have an UUID, since it's address string (e.g. ecash:qw82f...) is already an unique identifier. I noticed that when the codebase was already infected with too much `address.id` but have been avoiding it since then. The `Address.address` always fills up the same purpose but it's way more useful since it's information which is not exclusive to our databases but actually useful externally on the networks.

Reply:
thanks for the feedback, refactored